### ISSUE TITLE
Request: JAX

### ISSUE THREAD

**Issue by sreyan-ghosh** [simple-icons/simple-icons#13048]
### Brand Name

JAX

### Website

https://docs.jax.dev/en/latest/quickstart.html

### Popularity Metric

JAX has 1,828,146 weekly downloads on PyPi according to [PyPi Stats](https://pypistats.org/packages/jax)

### Official Resources for Icon and Color

https://docs.jax.dev/en/latest/_static/jax_logo_250px.png

**Comment by adamrusted** [comment]
The license on their repo does not grant trademark use - so we would need to request it. Could you reach out to them please @sreyan-ghosh?
